test/conformance/behaviors/sig-node/pod-spec.yaml

```diff
@@ -1,43 +1,47 @@
-- id: pod/basic-delete
+- id: pod/spec/basic-delete
   description: When a Pod resource is delete, the Pod containers must receive a TERM signal and the Pod MUST be deleted.
-- id: pod/hostname
+- id: pod/spec/hostname
   description: When the hostname field is set, a container running in the Pod MUST report the hostname as the specified value.
-- id: pod/subdomain
+- id: pod/spec/subdomain
   description: If specified, the fully qualified Pod hostname will be "<hostname>.<subdomain>.<pod
     namespace>.svc.<cluster domain>". If not specified, the pod will not have a
     domainname at all.
-- id: pod/terminationGracePeriodSeconds/in-spec
+- id: pod/spec/terminationGracePeriodSeconds/in-spec
   description: When the terminationGracePeriodSeconds is specified in the spec,
     processes running in the Pod MUST NOT receive a hard termination signal for at
     least that number of seconds after a delete request.
-- id: pod/terminationGracePeriodSeconds/in-delete
+- id: pod/spec/terminationGracePeriodSeconds/in-delete
   description: When the terminationGracePeriodSeconds is specified in a delete request,
     processes running in the Pod MUST NOT receive a hard termination signal for at
     least that number of seconds after the delete request.
-- id: pod/activeDeadlineSeconds
+- id: pod/spec/activeDeadlineSeconds
   description: Optional duration in seconds the pod may be active on the node relative
     to StartTime before the system will actively try to mark it failed and kill
     associated containers. Value must be a positive integer.
-- id: pod/hostNetwork/true
+- id: pod/spec/hostNetwork/true
   description: When hostNetwork is set to true, the Pod MUST use the host's network
     namespace.
-- id: pod/hostNetwork/false
+- id: pod/spec/hostNetwork/false
   description: When hostNetwork is set to false, the Pod MUST NOT use the host's network
     namespace.
-- id: pod/hostPID/true
+- id: pod/spec/hostPID/true
   description: When hostPID is set to true, the Pod MUST use the host's process
     namespace.
-- id: pod/hostPID/false
+- id: pod/spec/hostPID/false
   description: When hostPID is set to false, the Pod MUST NOT use the host's process
     namespace.
-- id: pod/hostIPC/true
+- id: pod/spec/hostIPC/true
   description: When hostIPC is set to true, the Pod MUST use the host's inter-process
     communication namespace.
-- id: pod/hostIPC/false
+- id: pod/spec/hostIPC/false
   description: When hostIPC is set to false, the Pod MUST NOT use the host's inter-process
     communication namespace.
+- id: pod/spec/basic-update
+  decription: Create a Pod with a unique label. Query for the Pod with the label as selector MUST be successful. Update the pod to change the value of the Label. Query for the Pod with the new value for the label MUST be successful.
+- id: pod/spec/container/resources
+  description: Create a Pod with CPU and Memory request and limits. Pod status MUST have QOSClass set to PodQOSGuaranteed.
```

Comment:
Ok, but does the test cover any of the specific logic around scheduling, eviction, etc. for guaranteed pods? If so we need behaviors for all of those. If not that's OK I suppose; those will show up later in sig-scheduling behaviors.

Reply:
This only checks for the corresponding status tags. I think the logic you mentioned should be deferred to sig-scheduling.